sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/ConfidentialLedgerClientLiveTests.cs

```diff
@@ -169,20 +166,19 @@ public async Task GetEnclaveQuotes()
         [RecordedTest]
         public async Task PostLedgerEntry()
         {
-            var result = await Client.PostLedgerEntryAsync(
-                RequestContent.Create(
-                    new { contents = Recording.GenerateAssetName("test") }));
+            var operation = await Client.PostLedgerEntryAsync(RequestContent.Create(new { contents = Recording.GenerateAssetName("test") }), waitForCompletion: true);
```

Comment:
just curious, if the default for `waitForCompletion` is always true, why do u pass it?

Reply:
is it because of LLC stuff?

Reply:
Partly - because the tests can see the internal overload, the arg is required to disambiguate the two.